Source/objects.cpp

```diff
@@ -2678,7 +2679,7 @@ bool OperateShrineGloomy(int pnum)
 	auto &player = Players[pnum];
 
 	// Increment armor class by 2 and decrements max damage by 1.
-	ForEachInventoryItem(player, [](ItemStruct &item) {
+	for (ItemStruct &item : PlayerItemsRange(player)) {

```

Comment:
```suggestion
	for (auto &item : PlayerItemsRange(player)) {
```

Reply:
Perhaps we shouldn't use `auto` here?

From https://google.github.io/styleguide/cppguide.html#Type_deduction

> Use type deduction only if it makes the code clearer to readers who aren't familiar with the project, or if it makes the code safer. Do not use it merely to avoid the inconvenience of writing an explicit type.

Reply:
Someone convinced me to always use `autio` in loops so I have tried doing so consistently :sweat_smile: